x/wasm-storage/types/events.go

```diff
@@ -3,7 +3,14 @@ package types
 const (
 	EventTypeStoreDataRequestWasm = "store_data_request_wasm"
 	EventTypeOverlayWasm          = "store_overlay_wasm"
+	EventTypeTallyCompletion      = "tally_completion"
+	EventTypeWasmExpiration       = "wasm_expiration"
 
-	AttributeHash     = "hash"
-	AttributeWasmType = "wasm_type"
+	AttributeWasmHash      = "wasm_hash"
+	AttributeWasmType      = "wasm_type"
+	AttributeRequestID     = "request_id"
```

Comment:
Sorry to keep coming back for small things but maybe we should make this `dr_id`? That's what the contract uses and consistency is nice 😬

Reply:
no problem. updated!